.github/workflows/chromatic.yaml

```diff
@@ -3,67 +3,44 @@
 # Workflow name
```

Comment:
These changes are pretty confusing, but this file is still here because of an earlier attempt to conserve snapshots before we knew about Turbosnap. Consider this file to be deleted (you can ignore the changes).